README.md

```diff
@@ -17,9 +17,33 @@ Request and perform code reviews from inside your IDE. Review any code, even if
 
 ## Setup
 
-After installing the theme, go to `Preferences | Appearance & Behavior | Appearance` and select one of the new themes in the theme dropdown. Once you have selected one of the theme options, click the **OK** button to apply the changes so you can start enjoying One Dark theme!
+After installing the theme, go to `Preferences | Appearance & Behavior | Appearance` and select the new `One Dark` theme in the theme dropdown. 
+Once you have selected `One Dark` in the theme options, click the **OK** button to apply the changes, so you can start enjoying One Dark theme!
 
-![Code example](https://github.com/one-dark/jetbrains-one-dark-theme/raw/master/docs/screenshots/configuration.png)
+![Code example](./docs/screenshots/theme-selection.png)
+
+## Configuration
+
+After set up, you can configure the appearance of the syntax highlighting of the theme.
+
+Go to `Preferences | Appearance & Behavior | One Dark Theme` where you have various appearance settings at your disposal.
+
+![Theme configurations](./docs/screenshots/theme-configuration.png)
```

Comment:
I'm not sure, but I think this has to point to the GitHub url so it shows properly in IntelliJ

Reply:
Ah! I did not realize it is being bundled into the plugin.